config/default/core.entity_view_display.node.localgov_event.default.yml

```diff
@@ -33,7 +34,12 @@ content:
     label: hidden
     settings: {  }
     third_party_settings: {  }
-    weight: 3
+    weight: 4
+    region: content
+  content_moderation_control:
```

Comment:
They're on quite a few display modes. I've asked Will on LP-22 how important this is, or if I can pick them up later. On a recent Intranet release, I removed them from loads of displays, and will do the same here as part of another piece of work.

Reply:
Are they there already, or have I introduced them? If I've introduced them, then we should remove them in this PR. If they are already there, but not visible to anon users because of permissions, then let's not let them hold up this specific PR.

Reply:
Judging by the live site, they are already there and not introduced by this PR so let's file a follow up issue to sort that part out.

Reply:
Great. What do we need to do now to get this merged/deployed to live so editors can start using it and I can start working on a follow up issue that depends on it?

Reply:
It's the same process as for PR 343 - merge into a release branch first, not directly to main.
If they can be deployed at the same time I suggest we create branch called _release/202312-01_ and merge both into that. Then I can deploy the release to Dev and Preprod.